addons/mail/static/src/components/activity/activity.js

```diff
@@ -2,10 +2,16 @@
 
 import { registerMessagingComponent } from '@mail/utils/messaging_component';
 
-const { Component } = owl;
+import Popover from "web.Popover";
+
+const { Component, markup } = owl;
 
 export class Activity extends Component {
 
+    get noteAsMarkup() {
+        return markup(this.activityView.activity.note);
+    }
```

Comment:
Completely agree with this but this should be a refactoring by the mail team, at the moment the place where the notes should be markuped is not obvious @seb-odoo told me they can be produced in multiple places and I'm not confident I can find all of them, there is also some places where the code assumes it deals with a string for comparison purposes (eg checking it it's truthy or comparing against `<p><br></p>` to check for emptiness). The same is true for the previous comment about prettyBody.

Would you be ok with merging this PR with these two as is, considering those were previously used in t-raw or inserted into the document by hand already, and let the mail team look into this post-merge?

Reply:
> Unfortunately (as far as I know) we currently don't have support for the server returning us "markup data" with certainty.
> 
> Ideally this is what should happen when the server is sending an HTML field such as `body` or `note`, the value sent on the bus or by RPC should not be a standard string but a markup one (or at least something that allows reconstructing it automatically directly on the arrival code in JS).

Absolutely not. In-band signaling is an attractive nuisance, emphasis very much on nuisance: it means every reflected structured user input (JSON, literals, some uses of query strings, ...) becomes a potential vector as the user can just swap a directly reflected string with a signaling value and voilà.

And it's completely unnecessary when it comes to model data, since the relevant information is already part of the model metadata.

Reply:
> Would you be ok with merging this PR with these two as is, considering those were previously used in t-raw or inserted into the document by hand already, and let the mail team look into this post-merge?

Sure, Is there a cross-project task to remember to look at these? Because not forgetting to *properly* fix them would be a good idea.

Reply:
Thank you!

I've created a task in the mail project: 2760839